africanlii/views/home.py

```diff
@@ -19,7 +19,12 @@ def get(self, request, *args, **kwargs):
         recent_documents = GenericDocument.objects.order_by("-date")[:5]
         recent_instruments = LegalInstrument.objects.order_by("-date")[:5]
         recent_legislation = Legislation.objects.order_by("-date")[:5]
-        documents_count = GenericDocument.objects.count()
+        documents_count = (
+            Judgment.objects.count()
+            + GenericDocument.objects.count()
+            + LegalInstrument.objects.count()
+            + Legislation.objects.count()
+        )
```

Comment:
You could consider using the CoreDocument model count instead. Its the base model that all others inherit.